inscriptions/ISic2928.xml

```diff
@@ -188,14 +188,13 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
                 <ab>
-                    <lb n="1"/>in hoc sipurgo deposita es<supplied reason="omitted">t</supplied> F<supplied reason="lost">un</supplied>dania
-                    <lb n="2"/>quae es<supplied reason="omitted">t</supplied> in pacae
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/>damia<gap reason="lost" extent="unknown" unit="character"/>
                 </ab>
             </div>
             <div type="apparatus">
                 <listApp>
                     <app>
-                        <note>Text after Ferrua 1989</note>
+                        <note>Text after Manganaro 1993</note>
```

Comment:
I would myself suggest text after Orsi 1907 [perhaps even add Manganaro 1993 'Damia' in error, or similar]; and add a note in the commentary that Ferrua's proposal (1989 no.93) to unite this with ISic2929 is to be rejected.